README.md

```diff
@@ -2,10 +2,56 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+It is the mission of the Node.js Foundation and the TSC to provide a home for 
+openly-governed and community-managed open source projects that are focused on 
+supporting and providing benefit to the Node.js ecosystem.
```

Comment:
IMO this is an even stronger statement than before that "the" only mission of the TSC and the Foundation is to grow the number of hosted projects. I was more comfortable with the previous version "The mission of the Node.js Foundation is to enable and encourage the adoption of Node.js".
That hosting any project that provides benefit to the Node.js ecosystems should be the mission of the TSC is still something that needs to be proven and qualified, in my opinion.